contrib/cookiecutter/ckan_extension/{{cookiecutter.project}}/ckanext/{{cookiecutter.project_shortname}}/cli.py

```diff
@@ -0,0 +1,20 @@
+import click
+
+__all__ = ['{{cookiecutter.project_shortname}}']
```

Comment:
Do we really need the `__all__` directives?

Reply:
No, I'm just adding it in advance. If `blanket`(#5169) will be merged into core, then we can just replace almost the whole body of `ExtPlugin` with few class-level decorators. But those `__all__` attributes have no functional meaning currently, so I see no problem in removing them